deploy-config/deploy-config-holesky.toml

```diff
@@ -25,17 +26,17 @@ min_withdrawals_batch_size = 1
 
 [dual_governance_config_provider]
 first_seal_rage_quit_support = 300
```

Comment:
For the basis points, maybe clearer to indent the fractional part with underscore:
```toml
first_seal_rage_quit_support = 3_00 # 3.00%
second_seal_rage_quit_support = 15_00 # 15.00%
```